ckanext/who_afro/templates/user/read_base.html

```diff
@@ -0,0 +1,26 @@
+{% ckan_extends %}
+
+
+{% block breadcrumb %}
+    {% if self.breadcrumb_content() | trim %}
+      <ol class="profile-page breadcrumb">
+        {% snippet 'snippets/home_breadcrumb_item.html' %}
+        {% block breadcrumb_content %}
+              <li><a href="{{ h.url_for('activity.dashboard') }}">{{ _('My account') }}</a></li>
+              <li class="active"><a>{{ _('My profile') }}</a></li>
+        {% endblock %}
+      </ol>
+    {% endif %}
+{% endblock %}
+
+{% block content_action %}
+  {% if h.check_access('user_update', user) %}
+    {% link_for _('Edit profile'), named_route='user.edit', id=user.name, class_='btn btn-default', icon='wrench' %}
+  {% endif %}
+{% endblock %}
```

Comment:
I actually think I prefer the default CKAN here where it says "Manage" - just to be consistent across all ckan objects. Sorry if that is different to what lewis has proposed.

Reply:
I have reverted to the default 'Manage' here : 
![image](https://github.com/fjelltopp/ckanext-who-afro/assets/15812707/3ec77029-d9ec-44d3-aa1e-595d111d4bdf)